README.md

```diff
@@ -213,4 +269,27 @@ public class MySlacker {
   // then just use the client!
 }
 ```
-```
+
+## Contributors
+ - [@szabowexler](https://github.com/szabowexler) [:computer:](https://github.com/HubSpot/slack-client/commits?author=szabowexler)
+ - [@zklapow](https://github.com/zklapow) [:computer:](https://github.com/HubSpot/slack-client/commits?author=zklapow)
+ - [@wsorenson](https://github.com/wsorenson) [:computer:](https://github.com/HubSpot/slack-client/commits?author=wsorenson)
+ - [@darcatron](https://github.com/darcatron) [:computer:](https://github.com/HubSpot/slack-client/commits?author=darcatron)
+ - [@dylanrb123](https://github.com/dylanrb123) [:computer:](https://github.com/HubSpot/slack-client/commits?author=dylanrb123)
+
+## License
```

Comment:
The full license is available at https://github.com/HubSpot/slack-client/blob/master/LICENSE. Do we actually need to include this? Seems like we could just link to that.

Reply:
Bigger ask: could you update https://github.com/HubSpot/slack-client/blob/master/LICENSE#L189?

Reply:
I can remove that how-to piece from the LICENSE file, but most OSS repos I've seen include the blurb it mentions in the readme

https://github.com/HubSpot/slack-client/blob/07ad49af36d9d1ae63805b050e54b45b552df906/LICENSE#L178-L187

Reply:
sure we can include it in the readme. It's in the repo, though, so I feel like we're covered.